src/ai/susi/mind/SusiAction.java

```diff
@@ -46,6 +46,7 @@
         websearch,     // do a web search on the client, show like rss rendering
         anchor,        // show/say a link
         map,           // show a map
+        alarm_set,     //set alarm
```

Comment:
Those are 2 different options that mobile phones offer. So I thought that it would be good to handle casses. Should I still change to timer?